### ISSUE TITLE
Add units to scale

### ISSUE THREAD

**Issue by codeofdusk** [skarkkai/touch-mapper#19]
I'm assuming that, for instance, 1:1800 means 1 cm on the map is 18 m on Earth. Could units please be added to the scale to clarify this?

**Comment by skarkkai** [comment]
Are there situations where 1:1800 scale could mean something else?

**Comment by codeofdusk** [comment]
I'm not sure – clarification on interpreting the scale would be useful somewhere (even if in the documentation).

**Comment by skarkkai** [comment]
I agree it wouldn't hurt to be a bit clearer about this. How about the UI saying something like "At this scale a map 20 cm across covers an area 480 meters across"?

**Comment by codeofdusk** [comment]
Sorry, somehow missed this.

Yes, that'd work well!

**Comment by skarkkai** [comment]
Done. Finally.
